### PR TITLE
Add phan, cleanup, and namespace

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: includes/MirahezeIRCRCFeedFormatter.php [hunk 1]
```diff
@@ -77,27 +82,12 @@ public function getLine( array $feed, RecentChange $rc, $actionComment ) {
...
-			if ( $attribs['rc_type'] == RC_NEW ) {
-				$query = '?oldid=' . $attribs['rc_this_oldid'];
-			} else {
-				$query = '?diff=' . $attribs['rc_this_oldid'] . '&oldid=' . $attribs['rc_last_oldid'];
-			}
-			if ( $useRCPatrol || ( $attribs['rc_type'] == RC_NEW && $useNPPatrol ) ) {
-				$query .= '&rcid=' . $attribs['rc_id'];
-			}
-
-			Hooks::runner()->onIRCLineURL( $url, $query, $rc );
```
Comment (redbluegreenhat) [inline includes/MirahezeIRCRCFeedFormatter.php:98]: I don't think we use this anywhere, but wouldn't be a bad idea to check.

**Review by redbluegreenhat — APPROVED** [review]
It took a while, wanted to confirm these methods existed (I'm not sure where the `->DisableSandbox` method comes from still. I assumed it was from the parent class but couldn't find it on the docs for UnboxedCommand. Still, the Command class uses it internally as well, so I assume it is good then). Overall, I like this PR. It brings MirahezeMagic more to the standards of the rest of our extensions, uses the `getNotifyUrl`  method on the IRCRCFeedFormatter (I didn't know that method of generating the URL was a copypaste from the `RecentChange` class).

Just one thing, since this PR removes a hook. Normally would block approval on that, but this extension is explicitly made only for Miraheze, it's not really that concerning.

File: includes/MirahezeIRCRCFeedFormatter.php (hunk 1 above)
Comment (redbluegreenhat) [reply includes/MirahezeIRCRCFeedFormatter.php:98]: Also, what is this way of running hooks? Does this even work nowadays?

Comment (Universal-Omega) [reply includes/MirahezeIRCRCFeedFormatter.php:98]: This hook is still executed it is just part of getNotifyUrl()

Comment (redbluegreenhat) [reply includes/MirahezeIRCRCFeedFormatter.php:98]: you're right. No concerns at all from me then.

**Pull request merged**